Formula/c/chdig.rb

```diff
@@ -0,0 +1,22 @@
+class Chdig < Formula
+  desc "Dig into ClickHouse with TUI interface"
+  homepage "https://github.com/azat/chdig"
+  url "https://github.com/azat/chdig/archive/refs/tags/v25.3.1.tar.gz"
+  sha256 "69fd0063279d4299d760a43a039ceb70064c0e8f24c9bec57c3924aaadcd4216"
+  license "MIT"
+  head "https://github.com/azat/chdig.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    generate_completions_from_executable(bin/"chdig", "--completions")
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/chdig --version")
+
+    assert_match "Error: Cannot connect to ClickHouse", shell_output("#{bin}/chdig dictionaries")
```

Comment:
```suggestion
```

Or if you really want to preserve it

```suggestion

    assert_match "Error: Cannot connect to ClickHouse", shell_output("#{bin}/chdig --url 255.255.255.255 dictionaries")
```

Reply:
yeah, I can go with either way.

Reply:
I am voting for removing this check completely.

Reply:
Once there will be tests I will submit changes to formula.

Reply:
(we need something like this to make sure the binary homebrew builds works fine)